r/R/chunked-array.R

```diff
@@ -75,23 +75,15 @@ ChunkedArray <- R6Class("ChunkedArray", inherit = ArrowObject,
       if (is.integer(i)) {
         i <- Array$create(i)
       }
-      # Invalid: Kernel does not support chunked array arguments
-      # so use the old method for both cases
-      if (inherits(i, "ChunkedArray")) {
-        return(shared_ptr(ChunkedArray, ChunkedArray__TakeChunked(self, i)))
-      }
-      assert_is(i, "Array")
-      return(shared_ptr(ChunkedArray, ChunkedArray__Take(self, i)))
+      # Invalid: Tried executing function with non-value type: ChunkedArray
```

Comment:
Is this comment (still) valid? Looks like this is calling `call_function` now

Reply:
Done in https://github.com/apache/arrow/pull/7308/commits/247a1048e85871214e0456f3ee347a9b3e5388da